client/tso_service_discovery.go

```diff
@@ -488,17 +488,15 @@ func (c *tsoServiceDiscovery) updateMember() error {
 		}
 	}
 
-	oldPrimary, primarySwitched, secondaryChanged :=
+	oldPrimary, primarySwitched, _ :=
```

Comment:
Do we still need to return the field if we don't use it? If we plan to use it, it's better to add a TODO here.

Reply:
We will use it after we add two watch loops to get get all registered tso servers and the entire election member list for every keyspace group so that every tso server has the global consistent view.
When we support followProxy, this returned value will be naturally used so no need to add TODO.